nix/nixos/cardano-tx-submitter-service.nix

```
@@ -0,0 +1,60 @@
+{ config, lib, pkgs, ... }:
+
+# notes:
+# this service exposes an http port, and connects to a cardano-node over a unix socket
+let
+  cfg = config.services.cardano-tx-submit;
+  envConfig = cfg.environment;
+  localPkgs = import ../. {};
+in {
+  options = {
+    services.cardano-tx-submit = {
+      enable = lib.mkEnableOption "enable the cardano-explorer tx submitter api";
```

For consistency this and the file name should be referred to as `cardano-tx-submit`. it's not an "explorer" component
```suggestion
      enable = lib.mkEnableOption "enable cardano-tx-submit";
```